docs/intelligentapps/modelconversion.md

```diff
@@ -0,0 +1,109 @@
+
+To use models from other catalogs, such as Azure AI Foundry or Hugging Face, in the AI Toolkit, you must first convert them to ONNX format.
+
+This tutorial guides you through converting Hugging Face models to ONNX format and loading them into the AI Toolkit.
+
+## Set up the environment
+
+To convert models from Hugging Face or Azure AI Foundry, you need the [Model Builder](https://onnxruntime.ai/docs/genai/howto/build-model.html) tool.
+
+Follow these steps to set up your environment:
+
+1. Ensure you have either [Anaconda](https://www.anaconda.com/download) or [Miniconda](https://www.anaconda.com/docs/getting-started/miniconda/install) installed on your device.
+
+2. Create a dedicated conda environment for Model Builder and install the necessary dependencies (`onnx`, `torch`, `onnxruntime_genai`, and `transformers`):
+
+    ```powershell
+    conda create -n model_builder python==3.11 -y
+    conda activate model_builder
+    pip install onnx torch onnxruntime_genai==0.6.0 transformers
+    ```
+
+    > Note: For certain newer models, such as Phi-4-mini, you may need to install the latest development version of transformers directly from GitHub:
+
+    ```powershell
+    pip install git+https://github.com/huggingface/transformers
+    ```
+
+## Access Hugging Face models
+
+There are multiple ways to access Hugging Face models. In this documentation, we use the `huggingface_hub` CLI as an example to demonstrate managing a model repository.
+
+> Note: Ensure your Python environment is properly set up before proceeding.
+
+To download models from Hugging Face:
+
+1. [Install the CLI](https://huggingface.co/docs/huggingface_hub/main/en/guides/cli#getting-started):
+
+    ```powershell
+    pip install -U "huggingface_hub[cli]"
+    ```
+
+2. [Download the model repository](https://huggingface.co/docs/huggingface_hub/main/en/guides/cli#download-an-entire-repository).
+
+3. All files in the downloaded repository will be used during conversion.
+
+## Create the directory structure
+
+The AI Toolkit loads ONNX models from its working directory:
+
+* Windows: `%USERPROFILE%\.aitk\models`
+* Unix-like systems (macOS): `$HOME/.aitk/models`
+
+To ensure your models load correctly, create the required four-layer directory structure within the AI Toolkit's working directory. For example:
+
+```powershell
+mkdir C:\Users\Administrator\.aitk\models\microsoft\Phi-3.5-vision-instruct-onnx\cpu\phi3.5-cpu-int4-rtn-block-32
+```
+
+In this example, the four-layer directory structure is `microsoft\Phi-3.5-vision-instruct-onnx\cpu\phi3.5-cpu-int4-rtn-block-32`.
+
+> [!TIP]
+> **The naming of the four-layer directory structure is important.** Each directory layer corresponds to a specific system parameter: `$publisherName\$modelName\$runtime\$displayName`. The `$displayName` appears in the local model tree view at the top-left side of the extension. Use distinct `displayName` values for different models to avoid confusion.
```

Comment:
```suggestion
> The naming of the four-layer directory structure is important. Each directory layer corresponds with a specific system parameter: `$publisherName\$modelName\$runtime\$displayName`. The `$displayName` appears in the local model tree view at the top-left side of the extension. Use distinct `displayName` values for different models to avoid confusion.
```